### PR TITLE
Delivery slips

### PR DESCRIPTION
**Pull request by Saimis777**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: content/1.7/back-office/orders/delivery-slips/view-delivery-slips.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Input field for the date, by default has the current day entered. Has a tooltip: "Format: 2011-12-31 (inclusive)". When pressed opens a calendar to choose a date.
+
+2) To
+
+Input field for the date, by default has the current day entered. Has a tooltip: "Format: 2011-12-31 (inclusive)". When pressed opens a calendar to choose a date.
+
+### Call to action
+
+ - Generate PDF, when pressed, if no PDF was done on the selected time, will show the message: "No delivery slip was found for this period."
```
Comment (fabienvallon) [inline content/1.7/back-office/orders/delivery-slips/view-delivery-slips.md:24]: And when the PDF is generated ? 
Where do we explain how is generate the PDF ?

File: content/1.7/back-office/orders/delivery-slips/view-delivery-slips.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Delivery slip options
+
+3 Input felds are presented:
+
+1) Delivery prefix
+
+This is an input field for the delivery prefix, depending on the countries installed on the shop there will be am input ready in the field, if for example the default country in the shop is Germany, the input will be: "#DE". Has a tooltip: "Prefix used for delivery slips".
```
Comment (fabienvallon) [inline content/1.7/back-office/orders/delivery-slips/view-delivery-slips.md:32]: How does it works with multi-language BO ? 

The behavior looks weird to me (1.7.8.0)

For example 
- if i choose fr (french) I get "#LI" prefix 
- if I choose gb I get "#DE" prefix

**Review by fabienvallon — CHANGES_REQUESTED** [review]

File: content/1.7/back-office/orders/delivery-slips/view-delivery-slips.md (hunk 1 above)
Comment (Saimis777) [reply content/1.7/back-office/orders/delivery-slips/view-delivery-slips.md]: @fabienvallon should we describe the ones in the back office or the ones that the user can check in his FO account?

File: content/1.7/back-office/orders/delivery-slips/view-delivery-slips.md (hunk 2 above)
Comment (Saimis777) [reply content/1.7/back-office/orders/delivery-slips/view-delivery-slips.md:32]: image


My prefix does not change, should this be registered as a bug? It stays on DE even when switching. PS 1784 version

**Pull request merged**